src/cnc/models/providers/amazon/environment_collection.py

```diff
@@ -172,7 +179,7 @@ def get_secret_value(self, secret_id, version_id=None):
         return secret_string
 
     def generate_tf_assets(self, config_files_path, rendered_files_path):
-        if self.application.flavor == "lambda-lite":
+        if self.application.flavor in ["lambda-lite", "ecs"]:
```

Comment:
we should probably have some attribute like `collection.has_serverless_services` so we don't do this when it's not necessary